### DISCUSSION TITLE
how to change the background color when dragging the item?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [react-grid-layout/react-grid-layout#1912] [Q&A] [answered]
When I drag the item of react-grid-layout, there will show a pink background, how to change the color of it?

![image](https://github.com/react-grid-layout/react-grid-layout/assets/12727582/610f173e-a8d3-4311-9816-5d2f710daa09)

**Comment by JamesCovar** [comment, marked as answer]
You could change the background class property to customize the appearance of the element.

`.react-grid-item.react-grid-placeholder {
 background: blue;
}`

**Reply by muda2bir** [reply]
@JamesCovar It's not working in my app, I added these styles in my index.css file

**Reply by jonahallibone** [reply]
make sure it's imported in the correct order

**Reply by daffaharizal** [reply]
big thanks bro! that's works.

**Comment by Aftab2209** [comment]
.react-grid-placeholder {
   background-color : blue
}
